### PR TITLE
Create app

### PR DESCRIPTION
**Pull request by brahimdidi**
In this part I create a front end react app that does the following:
[Backend pull request](https://github.com/brahimdidi/hello-rails-backend/pull/1)
-Fetch data from rails API app
-Dispay a message

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const root = createRoot(container);
+
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+    <Router>
+      <App />
+    </Router>
+    </Provider>
+  </React.StrictMode>
```
Comment (alexander16108) [inline src/index.js:20]: You should provide a link to the Rails API (backend app) in the README file for the front end. do this and your project is good to go 👍🏾 💯

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @brahimdidi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so close to making it on your first try but however there is still a good bit you have to do to call this project a success 😄 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (brahimdidi) [reply src/index.js:20]: you approved that one but not this one lol you got me confused man

**Review by omarsalem7 — APPROVED** [review]
Hi @brahimdidi 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**